lib/App/git/ship/perl.pm

```diff
@@ -557,6 +585,10 @@ __DATA__
 /META*
 /MYMETA*
 /pm_to_blib
+@@ .travis.yml
+# Enable Travis Continuous Integration at https://travis-ci.org
+# Learn more https://docs.travis-ci.com
+<%= $_[0]->_travis_configurated %>
```

Comment:
How about this?

```
---
language: perl
sudo: false
perl: [ <%= join ",", @{$_[0]->config->{test_perl_versions} || ["5.10", "5.20"]} %> ]
install:
  - "cpanm -n Devel::Cover Test::Pod Test::Pod::Coverage"
  - "cpanm -n --quiet --installdeps --with-develop ."
after_success: "cover -test -report coveralls"
notifications:
  email: false
```

Reply:
`config` cannot parse values into an array unless I'm missing something.